templates/semantic-ui/zenodo_rdm/macros/record_item.html

```diff
@@ -67,6 +68,12 @@
         {{ description | truncate(length=350, end='...') }}
       </p>
 
+      {# Communities list #}
+      <h5>
+        {% set communities_entries = record.parent.communities.entries %}
+        <b>{{ _("Part of:")}} {{ communities_list(communities=communities_entries) }}</b>
```

Comment:
Spoke with Lars and decided to use "Part of" in stead of "Published in", because this phrasing is already used in displaying Journal info (see screenshots)